data/mods/Magiclysm/items/enchanted_cloaks.json

```
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "Wildshape Cloak: Fish",

```

```suggestion
    "id": "wildshape_cloak_fish",

```
these are ids, and thus never player-facing. conventions are snake_case with no capitalization

---

Thank you KorGgenT, I will make sure to do this properly next time.